fab/operations/offline.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from fabric.api import env, local
+from fab.utils import generate_bower_command
+from fab.const import (
+    OFFLINE_STAGING_DIR,
+    WHEELS_ZIP_NAME,
+    HQ_ZIP_NAME,
+    BOWER_ZIP_NAME,
+    NPM_ZIP_NAME,
+)
+
+
+def prepare_zipfiles():
+    local(
+        'git clone --depth 1 --recursive https://github.com/dimagi/commcare-hq.git {}/commcare-hq'
+        .format(OFFLINE_STAGING_DIR)
+    )
+
+    # After we've created our HQ code zip, let's get bower and npm
```

Comment:
i talked about this a little with simon. i forgot the reasons why i convinced myself that was hard, maybe that was before i realized we could still git push to a remote repo. one difficulty would just be setting up that git server without internet access. couldn't do it with ansible, but perhaps it's ok to just hack something up